src/Commands/ContinuousIntegration/TravisCi/TravisCiQuestionProxy.php

```diff
@@ -0,0 +1,76 @@
+<?php
+
+namespace QuickStrap\Commands\ContinuousIntegration\TravisCi;
+
+use Symfony\Component\Console\Helper\QuestionHelper;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Console\Question\ConfirmationQuestion;
+use Symfony\Component\Console\Question\Question;
+
+class TravisCiQuestionProxy
```

Comment:
This class is not well named, it is not a proxy. 

It is more closely aligned to an extension of QuestionHelper. Since it is specific to travisCi, I wouldn't register as a default question helper but I would provide it the command constructor when the command is registered. `$defaultCommands[] = new TravisCiCommand(new TravisCiQuestionHelper);`